src/translations/en.json

```diff
@@ -17,8 +17,6 @@
   "ONBOARDING_CONNECT_DD_TRUSTLY_MODAL_TITLE": "Set up payment",
   "ONBOARDING_CONNECT_DD_TRUSTLY_MODAL_FAIL_BODY": "Something went wrong",
   "ONBOARDING_CONNECT_DD_TRUSTLY_MODAL_FAIL_CTA": "Try again",
-  "ONBOARDING_DOWNLOAD_PRE_HEADLINE": "Welcome!",
-  "ONBOARDING_DOWNLOAD_HEADLINE": "Download Hedvig to get started!",
```

Comment:
Just wanted to mention that we need to deploy the changes in my PR at the same time if we don't want missing text keys on `/download` page.

Reply:
Oh my bad, should have only commited the relevant translations, let's see if I can fix that :)

Reply:
No but it's tricky the way we use Lokalise now - especially when you remove or rename text keys.. 🤔 If you don't immediately merge those changes then the risk is really high that this happens. One way of doing it would be to not remove any old text keys until the new ones are merged. But then there's the risk of forgetting to do that and as a result having a bunch of unused text keys in Lokalise and in our translations files. 🤷‍♀️